mkdocs.yml

```diff
@@ -0,0 +1,102 @@
+site_name: AIAgents4Pharma
+
+nav:
+  - Home:
+      - Home : home.md
+  - Talk2Biomodels : 
+      - Talk2Biomodels: talk2biomodels/models/intro.md
+      - Models :
+        - System Bio Model: talk2biomodels/models/sys_bio_model.md
+        - Basico Model : talk2biomodels/models/basico_model.md 
+      - Tools :
+        - Asking question : talk2biomodels/tools/ask_question.md  
+        - Model Description : talk2biomodels/tools/model_description.md 
+        - Plotting Figure : talk2biomodels/tools/plot_figure.md  
+        - Simulate Model : talk2biomodels/tools/simulate_model.md         
+
+  - Talk2cells :
+      - Talk2cells: talk2cells/intro.md
+
+  - Talk2knowledgegraphs :
+      - Talk2knowledgegraphs: Talk2KnowledgeGraphs/intro.md 
+
+  - Documentation : 
+    - Ops :      
+        - Ops : Documentation_intro.md    
+        - CodeOps : ops/CodeOps.md
+        - DevOps : ops/DevOps.md   
+    - Contributing :
+        - Contributing : CONTRIBUTING.md    
+
+repo_url: https://github.com/VirtualPatientEngine/AIAgents4Pharma
+repo_name: AIAgents4Pharma
+
+theme:
+  logo: ./robot.png
+  #logo: 🤖
```

Comment:
This line can be deleted.

Reply:
Hi Douglas, I have removed all the redundant lines